pkg/services/sqlstore/sqlstore.go

```diff
@@ -294,7 +296,7 @@ type ITestDB interface {
 func InitTestDB(t ITestDB) *SqlStore {
 	t.Helper()
 	sqlstore := &SqlStore{}
-	sqlstore.skipEnsureAdmin = true
+	sqlstore.skipEnsureAdmin = false
```

Comment:
Think the test fails because you'll need to set `setting.DisableAdminUser` to true for the `sqlstore.Cfg` a couple of lines below.

Reply:
It was supposed to be true. I meant to change it back and put the wrong val.